MAVProxy/modules/mavproxy_adsb.py

```diff
@@ -11,26 +11,44 @@
 from pymavlink import mavutil
 
 obc_icons = {
-    0 : 'greenplane.png',
-    1 : 'greenplane.png',
-    2 : 'cloud.png',
-    3 : 'migbird.png',
-    4 : 'hawk.png'
+    100 : 'greenplane.png',
+    101 : 'greenplane.png',
+    102 : 'cloud.png',
+    103 : 'migbird.png',
+    104 : 'hawk.png'
 }
 
 obc_radius = {
-    0 : 300,
-    1 : 300,
-    2 : 173,
-    3 : 100,
-    4 : 200
+    100 : 300,
+    101 : 300,
+    102 : 173,
+    103 : 100,
+    104 : 200
 }
     
-def get_threat_radius(obc_type):
-    '''get threat radius for an OBC item'''
-    if obc_type in obc_radius:
-        return obc_radius[obc_type]
-    return 0
+def get_threat_radius(type_local, squawk_local):
+
+    if type_local == 255:
+        ''' objectAvoidance Database item, squawk contains radius in cm'''
+        return squawk_local * 0.01
+    elif type_local in obc_radius:
```

Comment:
pointless `elif`

Reply:
This block can be replaced with `return obc_radius.get(type_local, 0);` - but that structure is pre-existing so it can be left as-is

Reply:
done